server/raft.go

```diff
@@ -4066,6 +4066,7 @@ func (n *raft) processVoteRequest(vr *voteRequest) error {
 		n.term = vr.term
 		n.vote = noVote
 		n.writeTermVote()
+		n.cancelCatchup()
```

Comment:
It's minor, but we could move this to before the `n.term =` line, such that it happens before `n.writeTermVote()` (which does I/Os and therefore slower).

Reply:
Done